types/index.ts

```diff
@@ -1,5 +1,14 @@
+// By default, Sequelize automatically adds the fields createdAt and updatedAt to every model,
+// so they are also present in these types
+
```

Comment:
I don't understand this comment, since these types are not present here. I'm not sure if they are needed anyway. Can we remove this comment?

Reply:
I meant that, since we use these types to typehint objects that come from the database, they will also have these fields. I wasn't sure if I should add the fields to every type, because it would be a lot of repeated code. In any case, I removed the comment